tests/exceptions/exceptions_make_error_code.cpp

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for sycl::make_error_code function
+//
+*******************************************************************************/
+
+#include "exceptions.h"
+
+#define TEST_NAME exceptions_make_error_code
+
+namespace TEST_NAMESPACE {
+
+using namespace sycl_cts;
+
+/** @brief Provide verification for sycl::make_error_code function vorking
+ *  @param err_c Error code from sycl::errc enumeration
+ *  @param log sycl_cts::util::logger class object
+ */
+void check_sycl_working(sycl::errc err_c, util::logger &log) {
+  auto make_err_c_result{sycl::make_error_code(err_c)};
```

Comment:
Is it a good use for `auto`? I think the test must check the return type for the tested SYCL function.
```suggestion
  std::error_code make_err_c_result{sycl::make_error_code(err_c)};
```

Reply:
Alexey, can i skip this comment due to that this check is performed below?

Reply:
I'm okay if you move the type check right after the variable definition. Otherwise it's hard to follow the logic of the test.

Reply:
I've remove using `CHECK_TYPE` macros and use determinated type for `make_err_c_result` variable
I've removed `CHECK_TYPE` macros due to duplicated verification unnecessary
I've do this in this commit: 958080e51bc6720db7a68642eee3f0038a1947d4